Spoj1Divisor Summation/Program.cs

```diff
@@ -7,26 +7,49 @@
 
 namespace Spoj1Divisor_Summation
 {
-    public class Program
-    {
-        public static void Main()
-        {
-            var reader = new StreamReader(Console.OpenStandardInput());
-            var writer = new StreamWriter(Console.OpenStandardOutput());
+    public class Program
+    {
+        public static void Main()
+        {
+            var reader = new StreamReader(Console.OpenStandardInput());
+            var writer = new StreamWriter(Console.OpenStandardOutput());
             writer.AutoFlush = false;
-
-            DivisorSumation(reader, writer);
+            DivisorSumation(reader, writer);
+            Console.ReadLine();
         }
-
         public static void DivisorSumation(TextReader reader, TextWriter writer)
         {
-           //*******************************
-           //* TODO: Implement method body *
-           //*******************************
+            LiczbaPowtorzenObliczen(reader, writer);
+                reader.Close();
+                writer.Flush();
+                writer.Close();
 
-            reader.Close();
-            writer.Flush();
-            writer.Close();
         }
+        public static void LiczbaPowtorzenObliczen(TextReader reader, TextWriter writer)
+        {
+            int IloscLiczb = Convert.ToInt32(reader.ReadLine());
+            for (int i = 0; i < IloscLiczb; i++)
+            {
+                DodaniePodzielnikow(reader, writer);
+            }
+        }
+        public static void DodaniePodzielnikow(TextReader reader, TextWriter writer)
+        {
+            int input = Convert.ToInt32(reader.ReadLine());

```

Comment:
jeżeli już trzymasz się konwencji że piszesz polskie nazwy, to trzeba by konsekwentnie nazwać tę zmienną np "liczba"